### PR TITLE
fix docker compose fail

### PR DESCRIPTION
**Pull request by Mansh05**
This issue actually tries to solve docker compose issue. Please dont merge this before testing.

Here there are two things to keep in mind. We might need to update docker-compose file image value after testing. 
The ruby version will be bumped to 3.0.3 from 2.7.4. It should not hamper much since the CI/CD validates that the repo works with ruby  3 as well.

### CODE REVIEW

**Comment by weex** [comment]
Nice job! Tested this with one change (keeping postgres at version 9.6) and it works!

Postgres 9.6 is EOL so I'll create a separate issue to handle that. Upstream has a nice upgrade process which I'll link.

If you can modify this so that the postgres line is unchanged, I'll merge it and we can move on!

**Comment by Mansh05** [comment]
> Nice job! Tested this with one change (keeping postgres at version 9.6) and it works!
> 
> Postgres 9.6 is EOL so I'll create a separate issue to handle that. Upstream has a nice upgrade process which I'll link.
> 
> If you can modify this so that the postgres line is unchanged, I'll merge it and we can move on!

Oh great to hear. I will make the changes then.

File: docker-compose.yml [hunk 1]
```diff
@@ -57,13 +58,12 @@ services:
...
       - db
       - redis
-#      - es
+    #      - es
     volumes:
       - ./public/system:/mastodon/public/system
 
   streaming:
     build: .
-    image: dsterry/ecko
```
Comment (Mansh05) [inline docker-compose.yml:66]: @weex I removed these as well, Do you want me to put this in the same docker image or would you like it to be from the source only?

Comment (weex) [reply docker-compose.yml:66]: Thinking of what's easiest for a new installation, I would prefer to keep the `image:` line in.

**Comment by Mansh05** [comment]
@weex Made the postgres to 9.6. Please check

**Comment by weex** [comment]
LGTM!

**Pull request merged**